phpBB/phpbb/di/extension/tables.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\di\extension;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+/**
+ * Container tables extension
+ */
+class tables extends Extension
+{
+	/**
+	 * Loads a specific configuration.
+	 *
+	 * @param array            $configs   An array of configuration values
+	 * @param ContainerBuilder $container A ContainerBuilder instance
+	 *
+	 * @throws \InvalidArgumentException When provided tag is not defined in this extension
+	 */
+	public function load(array $configs, ContainerBuilder $container)
+	{
+		if (!$container->hasParameter('tables'))
+			return;
```

Comment:
No brackets here.

Reply:
sry, used to bad code at work :P

Reply:
Though this is pretty much the only place where I think this is valid thing to do.

Reply:
Thats fine then aswell, perhaps comment it stating that.